modules/server/main.tf

```diff
@@ -192,6 +192,23 @@ resource "google_compute_firewall" "forseti-server-allow-grpc" {
   depends_on = [null_resource.services-dependency]
 }
 
+resource "google_compute_firewall" "forseti-server-allow-sql-connection" {
```

Comment:
Might be good to have users choose to opt into this firewall rule by creating a separate variable to enable this rule and add a count of 0 or 1 based on that variable.

Reply:
@gkowalski-google was leaning towards putting this firewall to the customer config where this module is consumed.
@gkowalski-google, do you think gating this firewall rule with a new attribute suggested by @hshin-g is ok? If not, I can discard this PR, and send another one to surface `forseti-cloudsql-instance-ip` all the way to the top level as it will be needed by consumer of forseti module to create the firewall rule dynamically.

Reply:
friend ping~ @gkowalski-google wdyt?

Reply:
Hi @xingao267, sorry for the delay. We were having some internal discussion about this. I think adding the Cloud SQL IP as an output is the preferred way to go. Since Forseti does not need this to operate, and we want the bare minimum configuration to be included in this repo to allow Forseti to work.

Reply:
No worries. Sounds good. I'll modify this PR accordingly. @choonchernlim FYI.